include/enum_quda_fortran.h

```diff
@@ -126,9 +126,12 @@
 #define QUDA_INVALID_INVERTER QUDA_INVALID_ENUM
 
 #define QudaEigType integer(4)
-#define QUDA_LANCZOS 0 //Normal Lanczos eigen solver
-#define QUDA_IMP_RST_LANCZOS 1 //implicit restarted lanczos solver
-#define QUDA_INVALID_TYPE QUDA_INVALID_ENUM
+#define QUDA_LANCZOS 0           // Lanczos solver
+#define QUDA_IMP_RST_LANCZOS 1   // Implicitly restarted lanczos solver
+#define QUDA_THICK_RST_LANCZOS 2 // Thick restarted lanczos solver
+#define QUDA_ARNOLDI 3           // Arnoldi solver
+#define QUDA_IMP_RST_ARNOLDI 4   // Implicitly restarted Arnoldi solver
```

Comment:
Do we need the `SPECTRUM` enums defined in the Fortran interface?

Reply:
I don't know, I just put them there because the the old enums were there. Should they go?

Reply:
For now I would suggest putting the `QUDA_SPECTRUM_XXX` defines in the enum_fortran_fortan.h file as well, for consistency if nothing else.  Eventual plan is to auto generate the `enum_quda_fortran.h` file from the the `enum_quda.h` one once I figure a good way to do that....